lib/autoload/filename_sanitizer.rb

```diff
@@ -0,0 +1,8 @@
+module FilenameSanitizer
```

Comment:
No, we have to consider it invalid. This is because if we zip up that file and the person downloading is on windows, he can't extract the zip.